test/basic.test.js

```diff
@@ -354,7 +354,7 @@ test('basic prettifier tests', (t) => {
         cb()
       }
     })
-    const prettyStream = prettyFactory({useMetadata: true, outputStream: dest})
+    const prettyStream = prettyFactory().asMetaWrapper(dest)
```

Comment:
I'm confused here. How will options like `colorize` be recognized?

Reply:
Oh, I see how it works now. Can you add a simple unit test to show options being passed in? I think `levelFirst` would be sufficient.